config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Spree::Core::Engine.routes.draw do
   get 'r/:code' => 'reffiliate#referral', as: 'referral'
-  get 'a/:path' => 'reffiliate#affiliate', as: 'affiliate'
+  get 'partner/:path' => 'reffiliate#affiliate', as: 'affiliate'
```

Comment:
@SAkshat i don't think we are using these routes.

Reply:
@siddharth28 - Removed unused routes. Have kept `/partner/:path` as we are using url helpers in the code.